### PR TITLE
Implement client-server connection

### PR DESCRIPTION
**Pull request by JadenBalogh**

### CODE REVIEW

File: client/src/components/About.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+            <img class='round-20' src={teamImg} />
+          </div>
+          <div class="flex-grow-1 pl-20">
+            <h3>Our History...</h3>
+            
+              <p>
+              As an established and ambitious international Management School, research is at the heart of our success and informs our teaching, pushes boundaries and transforms lives. Through our collaborative approach and cutting edge knowledge we strive to be a beacon for top talent.
+              </p>
+              <p>
+              As an established and ambitious international Management School, research is at the heart of our success and informs our teaching, pushes boundaries and transforms lives. Through our collaborative approach and cutting edge knowledge we strive to be a beacon for top talent.
```
Comment (JadenBalogh) [inline client/src/components/About.js:37]: This all seems placeholder. Make sure in the future you add content that is specific to our project's requirements. I'll update this for this week as part of my review.

File: Documentation/Dev/Database Design - Milestone 1.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+## Table & Column Naming Conventions:
+- Tables always start with a capital, have no spaces, and are camel cased. eg: "SearchableInfo" (Upper Camel Case aka Pascal Case) 
+- Tables that are intended to store additional information to be referenced by an ID in another table have the postfix "Type". eg: "GenderType"
+- Columns start with a lower case letter and are camel cased eg: "birthYear" (lower Camel Case aka Dromedary Case)
+
+## Member
+### memberID - Auto-Incrementing Primary Key
+Type: INT
+
```
Comment (Jaredhw) [inline Documentation/Dev/Database Design - Milestone 1.md:14]: This all needs to be updated, I might remove it from here and add it to gitignore and keep it local until it's up to date so nobody gets confused.

**Review by aKrisnanto — APPROVED** [review]
Looks good, will need to discuss how the back-end connection works to work on the project properly

File: client/src/components/About.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+// About page
+class About extends Component {
+  render() {
+    return (
+      <div class="about">
+        <h1>Welcome To HomeTogether.ca</h1>
+        <div class="flex flex-justify-between flex-item-center border p-20 round-20 hover-shadow">
+          <div class="flex-grow-1 pr-20">
+            <h3>Our Purpose...</h3>
```
Comment (Jaredhw) [inline client/src/components/About.js:14]: I like the use of flex-boxes! 👍

**Review by Jaredhw — APPROVED** [review]

File: client/src/components/Main.js [hunk 4]
```diff
@@ -11,6 +13,8 @@ class Main extends Component {
       <Switch>
         <Route exact path='/' component={Home} />
         <Route exact path='/members' component={Members} />
+        <Route exact path='/listings' component={Listings} />
+        <Route exact path='/About' component={About} />
```
Comment (JadenBalogh) [inline client/src/components/Main.js:17]: path should be lowercase to match the other entries

File: client/src/components/NavBar.js [hunk 5]
```diff
@@ -7,9 +7,49 @@ class NavBar extends Component {
...
+              <li><NavLink class='flex flex-justify-between flex-item-center' to='/about'><span>Classes, Clubs +</span><span>&gt;</span></NavLink>
+                <ul>
+                  <li><NavLink class='flex flex-justify-between flex-item-center' to='/about'><span class="flex-grow-1">LawnDoers</span><span>lawndoers.com</span><span>1-800-345-6543</span></NavLink></li>
+                  <li><NavLink class='flex flex-justify-between flex-item-center' to='/about'><span class="flex-grow-1">LawnDoers</span><span>lawndoers.com</span><span>1-800-345-6543</span></NavLink></li>
+                  <li><NavLink class='flex flex-justify-between flex-item-center' to='/about'><span class="flex-grow-1">LawnDoers</span><span>lawndoers.com</span><span>1-800-345-6543</span></NavLink></li>
+                  <li><NavLink class='flex flex-justify-between flex-item-center' to='/about'><span class="flex-grow-1">LawnDoers</span><span>lawndoers.com</span><span>1-800-345-6543</span></NavLink></li>
+                  <li><NavLink class='flex flex-justify-between flex-item-center' to='/about'><span class="flex-grow-1">LawnDoers</span><span>lawndoers.com</span><span>1-800-345-6543</span></NavLink></li>
```
Comment (JadenBalogh) [inline client/src/components/NavBar.js:48]: Use CSS for styling

File: client/src/components/About.js (hunk 3 above)
Comment (AA666666) [reply client/src/components/About.js:14]: Thanks Jared😄

**Comment by AA666666** [comment]
Thanks for you guys' comments

**Pull request merged**